feature-libs/product-multi-dimensional/selector/components/selector/product-multi-dimensional-selector.component.html

```diff
@@ -27,7 +27,7 @@
             <option
               *ngFor="let option of category.variantOptions"
               [value]="option.code"
-              [selected]="isSelected(option.code)"
+              [selected]="isSelected(option.code, product)"
```

Comment:
`query:` is this component often re-rendered? or it's rather destroyed and recreated? I'm asking in terms of invoking a method in the template - we have to remember that the method is going to be called for each element each time this component re-renders.

Reply:
Every time the user changes the variant, it will trigger a new page load. Additionally, invoking a method from the template in this case has a marginal impact on performance.